src/main/scala/ch/epfl/bluebrain/nexus/admin/projects/Projects.scala

```diff
@@ -46,6 +45,9 @@ class Projects[F[_]](agg: Agg[F], index: ProjectCache[F], organizations: Organiz
     retryStrategy: RetryStrategy[F]
 ) {
 
+  private def invalidIriGeneration(organization: String, label: String, param: String): String =
+    s"the value of the project's '$param' could not be generated properly from the provided project '$organization/$label'"
```

Comment:
Is this even possible given the fact that the `$organization/$label` is part of the URL by which project is accesed?

Reply:
thats a good point...